### PR TITLE
Format only rst files with blackdoc

### PR DESCRIPTION
**Pull request by beroda**
### Overview

See comment here https://github.com/pyvista/pyvista/pull/6991#issuecomment-2547894606

### CODE REVIEW

File: doc/source/api/utilities/utilities.rst [hunk 1]
```diff
@@ -131,28 +131,21 @@ PyVista Version Information
...
-
-   >>> import pyvista
-   >>> pyvista.vtk_version_info
-   VTKVersionInfo(major=9, minor=1, micro=0)
-
-   Get the major version of VTK
+>>> # Output the version of VTK.
+>>> import pyvista
+>>> pyvista.vtk_version_info
+VTKVersionInfo(major=9, minor=1, micro=0)
```
Comment (beroda) [inline doc/source/api/utilities/utilities.rst:147]: Was a mix of doctest and python code block. Need to see how it looks like in the documentation build

Comment (beroda) [reply doc/source/api/utilities/utilities.rst:147]: Looks fine in the documentation artifacts (left is previous version, right is current).

image

**Review by user27182 — APPROVED** [review]
LGTM, thanks!

**Review by tkoyama010 — APPROVED** [review]
Thanks!

**Pull request merged**